apps/addons/tests/test_commands.py

```diff
@@ -87,17 +87,32 @@ def test_approve_addons_get_files_bad_guid():
     assert approve_addons.get_files(['foo']) == [addon1_file]
 
 
+def id_function(fixture_value):
+    """Convert a param from the use_case fixture to a nicer name.
+
+    By default, the name (used in the test generated from the parameterized
+    fixture) will use the fixture name and a number.
+    Eg: test_foo[use_case0]
+
+    Providing explicit 'ids' (either as strings, or as a function) will use
+    those names instead. Here the name will be something like
+    test_foo[public-unreviewed-full], for the status values, and if the file is
+    unreviewed.
+    """
+    addon_status, file_status, review_type = fixture_value
+    return '{0}-{1}-{2}'.format(amo.STATUS_CHOICES_API[addon_status],
+                                amo.STATUS_CHOICES_API[file_status],
+                                review_type)
+
+
```

Comment:
This isn't related to this PR, it's just that I was fiddling with the fixtures and stuff, and thought it would be clearer that way. I can submit that in another PR if needed.

This change doesn't change the tests at all, only the way they are named.